src/tactic/command/pipeline_task_trigger.py

```diff
@@ -299,20 +299,46 @@ def execute(my):
 
         column = data.get('column')
         src_status = data.get('src_status')
+
         
-        task = my.get_caller()
-        if task.get_value("status") != src_status:
-            return
 
-        task.set_now(column)
-        task.commit()
+        item = my.get_caller()
 
 
+        if isinstance(item,SObject):
+            if not isinstance(item, Task):
```

Comment:
to make this more readable ,. remove the return statement.
change the condition to 
if isinstance(item, Task):

else: